apps/docs/src/app/core/component-docs/action-sheet/examples/action-sheet-mobile/action-sheet-mobile-example.component.html

```diff
@@ -1,7 +1,7 @@
 <div style="position: relative">
     <fd-action-sheet [mobile]="true">
         <fd-action-sheet-control>
-            <button fd-button glyph="settings"></button>
+            <button fd-button glyph="settings" title="Open Action Sheet Mobile" aria-haspopup="menu"></button>
```

Comment:
Great work! I was looking at this documentation: https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

and you need `aria-controls` for the button and an id for the menu it controls. You will need an `id` for the button as well, as you will need it for the `aria-labelledby` attr of the ul.